events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -779,184 +786,213 @@
+                                    category = "19";
+                                    break;
+                                case "Gadgets":
+                                    category = "30";
+                                    break;
+                                case "Art":
+                                    category = "25";
+                                    break;
+                                case "Celebrities":
+                                    category = "26";
+                                    break;
+                                case "Animals":
+                                    category = "27";
+                                    break;
+                                case "Vehicles":
+                                    category = "28";
+                                    break;
+                                case "Mythology":
+                                    category = "20";
+                                    break;
+                                case "Sports":
+                                    category = "21";
+                                    break;
+                                case "History":
+                                    category = "23";
+                                    break;
+                                case "Geography":
+                                    category = "22";
+                                    break;
+                                case "Politics":
+                                    category = "24";
+                                    break;
+                            }
+                            break;
+                        case 'newQuestion':
+                            getQuestion();
+                            break;
+                        case 'showQuestion':
+                            showQuestion();
+                            break;
+                        case 'showAnswers':
+                            showAnswers();
+                            startTimer();
+                            Script.setTimeout(function() {
+                                showCorrect();
+                                if (useGoogle) {
+                                    tablet.emitScriptEvent("re-enableCustom");
+                                } else {
+                                    tablet.emitScriptEvent("re-enable");
+                                }
+                            }, TEN_SECONDS_MS);
+                            break;
+                        default:
+                            print(JSON.stringify(event));
+                            print("error in detecting event.type");
+                    }
                 }
+                lastTime = newTime;
             }
         }
     }
 
     function onScreenChanged(type, url) {
         open = (url === appPage);
         button.editProperties({isActive: open});
+        if (open) {
+            tablet.webEventReceived.connect(onWebEventReceived);
```

Comment:
where is this signal disconnected, besides in `appEnding()`? if you don't disconnect it, you're going to run into problems

Reply:
disconnect moved to onClicked and triggers when closing the app.